starfish/stack.py

```diff
@@ -18,7 +18,7 @@
 from scipy.ndimage.filters import gaussian_filter
 from scipy.stats import scoreatpercentile
 from skimage import exposure
-from skimage import img_as_float, img_as_uint
+from skimage import img_as_float32, img_as_uint
```

Comment:
I realize you may hate me, but I think a prudent way of handling this is storing img_as_float32 and np.float32 as "constants" in `starfish.types`, and use that throughout the codebase.

Reply:
Yea, I could see this being useful. Good suggestion. :+1:

Reply:
#549